### PR TITLE
VE : Verification routine

### PR DESCRIPTION
**Pull request by BellezaEmporium**
- Use linkastream for 2 channels (one had a certain issue with the livestream)
- Add alternative links for TeleSUR
- TVes uses Opencaster, those links could be added if the company behind VLC adds an "origin" option method.
- As always, adaptive over chunklists

https://edge1o.live.opencaster.com/caster/erCkqDzdGzpj.m3u8 works if origin is set to https://www.opencaster.com

### CODE REVIEW

File: channels/ve.m3u [hunk 1]
```diff
@@ -1,51 +1,61 @@
...
 #EXTINF:-1 tvg-id="TVVENEZUELA.ve" tvg-name="TV VENEZUELA" tvg-country="VE" tvg-language="" tvg-logo="" group-title="",TV VENEZUELA (480p)
-http://d2fxrfbiedz1tm.cloudfront.net/livepaxtv/smil:PC.smil/chunklist_b1735000.m3u8
+http://d2fxrfbiedz1tm.cloudfront.net/livepaxtv/smil:PC.smil/playlist.m3u8
+#EXTINF:-1 tvg-id="ValeTV.ve" tvg-name="Vale TV" tvg-country="VE" tvg-language="" tvg-logo="https://www.lyngsat-stream.com/logo/tv/vv/vale_tv_ve.png" group-title="",Vale TV
+http://vcp1.myplaytv.com/valetv/valetv/playlist.m3u8
 #EXTINF:-1 tvg-id="VePlus.ve" tvg-name="Ve Plus" tvg-country="VE" tvg-language="" tvg-logo="" group-title="",Ve Plus (480p)
 http://190.122.96.187:8888/http/006
 #EXTINF:-1 tvg-id="Venevision.ve" tvg-name="Venevision" tvg-country="VE" tvg-language="" tvg-logo="" group-title="",Venevision
-http://177.52.221.214:8000/play/a0cw/index.m3u8
+https://linkastream.co/headless?url=https://www.youtube.com/watch?v=YNNR0vDH6vA
```
Comment (freearhey) [inline channels/ve.m3u:55]: Doesn't work.
image

File: channels/ve.m3u [hunk 2]
```diff
@@ -1,51 +1,61 @@
...
 #EXTINF:-1 tvg-id="VenezolanadeTelevision.ve" tvg-name="Venezolana de Televisión" tvg-country="VE" tvg-language="" tvg-logo="" group-title="",Venezolana de Televisión
 http://177.52.221.214:8000/play/a0cj/index.m3u8
 #EXTINF:-1 tvg-id="VenezolanadeTelevision.ve" tvg-name="Venezolana de Televisión" tvg-country="VE" tvg-language="" tvg-logo="" group-title="",Venezolana de Televisión
-https://livestream.streamingconnect.com/hls/vtvweb.m3u8
-#EXTINF:-1 tvg-id="VepacoTV.ve" tvg-name="Vepaco TV" tvg-country="VE" tvg-language="" tvg-logo="" group-title="",Vepaco TV [Not 24/7]
-http://vcp1.myplaytv.com:1935/tvepaco/tvepaco/chunklist_w2073389661.m3u8
+https://linkastream.co/headless?url=https://dai.ly/x828i6j
```
Comment (freearhey) [inline channels/ve.m3u:59]: image

File: channels/ve.m3u [hunk 3]
```diff
@@ -1,51 +1,61 @@
 #EXTM3U
-#EXTINF:-1 tvg-id="123TV.ve" tvg-name="123 TV" tvg-country="VE" tvg-language="Spanish" tvg-logo="" group-title="",123 TV
```
Comment (freearhey) [inline channels/ve.m3u:2]: Maybe it would be better to add `[Offline]` to leave a description in the playlist?

File: channels/ve.m3u (hunk 2 above)
Comment (freearhey) [reply channels/ve.m3u:59]: I don't know if it's just me, but not a single link from `linkastream.co` has worked for me in the last month. Is it different for you?

**Review by freearhey — CHANGES_REQUESTED** [review]

File: channels/ve.m3u (hunk 2 above)
Comment (LaneSh4d0w) [reply channels/ve.m3u:59]: Same for me, Keystroke told me that his server doesn't seem to handle the amount of requests it gets. He's finding solutions.

File: channels/ve.m3u (hunk 3 above)
Comment (LaneSh4d0w) [reply channels/ve.m3u:2]: I'm quite unsure as if it'll come back online, as it's a Astra CP link.

**Review by freearhey — APPROVED** [review]

File: channels/ve.m3u (hunk 2 above)
Comment (freearhey) [reply channels/ve.m3u:59]: Okay.

**Pull request merged**